xml/geo_manage_i.xml

```diff
@@ -164,25 +197,115 @@ booth[27900]: 2014/08/13_10:21:23 info: revoke succeeded!</screen>
      stopped or demoted cleanly.
     </para>
    </step>
-   <step>
+  <!-- <step>
     <para>
      Revoke <literal>&ticket1;</literal> from site
      <literal>&cluster1;</literal> with:
     </para>
 <screen>&prompt.root;<command>booth</command> revoke -s 192.168.201.100 &ticket1;</screen>
-   </step>
+   </step>-->
    <step>
     <para>
-     After the ticket has been revoked from its original site, grant it to the
-     site <literal>&cluster2;</literal> with:
+     <!--After the ticket has been revoked from its original site, -->Grant
+     <literal>&ticket1;</literal> to the site <literal>&cluster2;</literal> with:
     </para>
 <screen>&prompt.root;<command>booth</command> grant -s 192.168.202.100 &ticket1;</screen>
    </step>
+   <step>
+    <para>
+     Activate <literal>&ticket1;</literal> on site <literal>&cluster1;</literal> with:
+    </para>
+<screen>&prompt.root;<command>booth</command> grant -a 192.168.202.100 &ticket1;</screen>
+     <para>Resources depending on &ticket1; will automatically fail over to
+       <literal>&cluster1;</literal> in case
+       <literal>&cluster2;</literal> fails.</para>
+   </step>
   </procedure>
-  <para>
-   <remark>taroth 2018-04-03: FATE#322100 - add how to proceed for manual failover
-    of tickets</remark>
-  </para>
- </sect1>
+ </sect2>
+ <sect2 xml:id="sec.ha.geo.manage.cli.booth.man-ticket.move">
+  <title>Moving a Manual Ticket</title>
+   <para>
+    Assuming that you want to manually move <literal>ticketnfs</literal> from
+    site <literal>&cluster1;</literal> (with the virtual IP
+    <literal>192.168.201.100</literal>) to site <literal>&cluster2;</literal>
+    (with the virtual IP <literal>192.168.202.100</literal>), proceed as
+    follows:
+   </para>
+  <procedure>
+   <step>
+    <para>Log in to <literal>&cluster1;</literal>.</para>
+   </step>
+   <step>
+    <para>
+     Set <literal>&ticket3;</literal> to standby with the following command:
+    </para>
+<screen>&prompt.root;<command>crm_ticket</command> -t &ticket3; -s</screen>
+   </step>
+   <step>
+    <para>
+     Wait for any resources that depend on <literal>&ticket3;</literal> to be
+     stopped or demoted cleanly.
+    </para>
+   </step>
+   <step>
+    <para>
+     Grant <literal>&ticket3;</literal>to the site <literal>&cluster2;</literal> with:
+    </para>
+<screen>&prompt.root;<command>booth</command> grant -s 192.168.202.100 -F &ticket3;</screen>
```

Comment:
Similarly, we should explicitly revoke the ticket first and then grant it again without "-F". Unnecessary "-F" would introduce double-granting (concurrency violation).